test/message.test.js

```diff
@@ -26,6 +26,22 @@ describe('Message tests', () => {
       done();
     });
   });
+  it('should detect if messagePriority is Normal, Urgent or Critical',
+  (done) => {
+    const message = {
+      messageBody: 'this is a message',
+      messagePriority: 'something'
+    };
+    supertest(app).post('/api/group/1/message')
+    .set('x-access-token', token)
+    .send(message)
+    .end((err, res) => {
+      assert.equal(res.statusCode, 400);
+      assert.equal(res.body.error,
+        'Validation error: Message priority can only Normal, Urgent, or Critical');
```

Comment:
I think there's a typo in this line. `Validation error: Message priority can only Normal, Urgent, or Critical` could be `Validation error: Message priority can only be Normal, Urgent, or Critical`

Reply:
Thanks. Changing it immediately.